docs/docs/reference/cli.md

```diff
@@ -79,7 +79,7 @@ constellation config generate {aws|azure|gcp|openstack|qemu|stackit} [flags]
 ```
   -a, --attestation string   attestation variant to use {aws-sev-snp|aws-nitro-tpm|azure-sev-snp|azure-trustedlaunch|gcp-sev-es|qemu-vtpm}. If not specified, the default for the cloud provider is used
   -h, --help                 help for generate
-  -k, --kubernetes string    Kubernetes version to use in format MAJOR.MINOR (default "v1.27")
+  -k, --kubernetes string    Kubernetes version to use in format MAJOR.MINOR (default "v1.27.4")
```

Comment:
You would need to check in with other devs here, but I think the idea was for users to not have to specify a patch version when using this flag

Reply:
That is indeed the point of the flag.

Reply:
fixed